docs/endpoints/actions.md

```diff
@@ -24,6 +24,7 @@ Example Response:
             "name": "august-2018-turbovote",
             "campaign_id": 9005,
             "post_type": "voter-reg",
+            "callpower_campaign_id": null,
```

Comment:
This is so nitpicky, but let's leave these as `null` since they're `post_type` of `voter-reg` and `share-social`. (Or, if we want to demonstrate how this field would look, let's change one of these actions' type to `phone-call`).

Reply:
Not nitpicky at all! This is great - we should have accurate documentation for less confusion! :)